PWGDQ/Core/VarManager.h

```diff
@@ -784,6 +786,36 @@ void VarManager::FillPair(T1 const& t1, T2 const& t2, float* values)
   }
 }
 
+template <typename T1, typename T2>
+void VarManager::FillPairMC(T1 const& t1, T2 const& t2, float* values, PairCandidateType pairType)
+{
+  if (!values) {
+    values = fgValues;
+  }
+
+  float m1 = fgkElectronMass;
+  float m2 = fgkElectronMass;
+  if (pairType == kJpsiToMuMu) {
+    m1 = fgkMuonMass;
+    m2 = fgkMuonMass;
+  }
+
+  if (pairType == kElectronMuon) {
+    m1 = fgkElectronMass;
```

Comment:
This assignment is not needed here, since its already done at l796

Reply:
removed